assets/src/scss/new-identity/style.scss

```diff
@@ -15,4 +15,23 @@
   --button-primary--color: var(--grey-900);
   --button-primary--visited--color: var(--grey-900);
   --button-primary--hover--color: var(--grey-900);
+  --button--focus--outline: 2px solid #99c3ff;
+  --button--focus--outline-offset: 2px;
+  --top-page-tags--tag-item--background: var(--gp-green-100);
+  --post-tag-button--font-weight: 700;
+  --post-tag-button--active--color: var(--grey-900);
+  --post-tag-button--active--background-color: var(--gp-green-200);
+  --post-tag-button--background-color: var(--gp-green-100);
+  --post-tag-button--border-color: var(--gp-green-100);
+  --post-tag-button--color: var(--grey-900);
+  --post-tag-button--hover--background-color: var(--gp-green-200);
+  --post-tag-button--hover--border-color: var(--gp-green-200);;
+  --post-tag-button--hover--color: var(--grey-900);
+  --post-tag-button--visited--color: var(--grey-900);
+}
+
+.post-tag-button {
```

Comment:
Maybe instead of defining this here, you could just add the hover border variable where we have the rest of the `post-tag-button` styles (so [here](https://github.com/greenpeace/planet4-master-theme/blob/af4d9553c7febd7d15a0403271acf26744f18f07/assets/src/scss/components/_buttons.scss#L285-L288)) and set it to `var(--link--color)`?

Reply:
No, it won't 🤔 I'm not saying you should change the `--link--color` variable, sorry if that was confusing. I'm just saying you could directly write in `_buttons.scss`:
```
&:hover {
      background-color: var(--link--color);
      border-color: var(--link--color);
      color: $white;
}
```
and that would create the `--post-tag-button--hover--border-color` that we can override here! Does it make sense?

Reply:
Yes, that really make sense!